examples/1_Santanilla_Nanomole_HTE/example_Santanilla.ipynb

```diff
@@ -5616,45 +5628,47 @@
     "\n",
     "# Create reaction products\n",
     "# Desired product\n",
     "product = outcome.products.add() \n",
-    "product.compound.identifiers.add().CopyFrom(\n",
-    "    reaction_pb2.CompoundIdentifier(value=r'placeholder', type='SMILES')\n",
-    ")\n",
+    "product.identifiers.add(value=r'placeholder', type='SMILES')\n",
     "product.is_desired_product = True\n",
-    "product.analysis_identity.append('UPLC')\n",
-    "product.analysis_yield.append('UPLC')\n",
+    "\n",
+    "product.measurements.add(type='IDENTITY', analysis_key='UPLC', uses_authentic_standard=True)\n",
+    "product.measurements.add(type='AREA', analysis_key='UPLC', uses_authentic_standard=True)\n",
+    "product.measurements.add(type='AREA', analysis_key='UPLC', uses_authentic_standard=True,\n",
+    "                         uses_internal_standard=True, is_normalized=True)\n",
     "\n",
     "# Remaining starting material (electrophile or ArBr)\n",
-    "electrophile = outcome.products.add() \n",
-    "electrophile.compound.identifiers.add().CopyFrom(\n",
-    "    reaction_pb2.CompoundIdentifier(value=r'placeholder', type='SMILES')\n",
-    ")\n",
-    "electrophile.is_desired_product = False\n",
-    "electrophile.analysis_identity.append('UPLC')\n",
-    "electrophile.analysis_yield.append('UPLC')\n",
+    "electrophile = outcome.products.add(is_desired_product=False) \n",
+    "electrophile.identifiers.add(value=r'placeholder', type='SMILES')\n",
+    "electrophile.measurements.add(type='IDENTITY', analysis_key='UPLC', uses_authentic_standard=True)\n",
+    "electrophile.measurements.add(type='AREA', analysis_key='UPLC', uses_authentic_standard=True)\n",
+    "electrophile.measurements.add(type='AREA', analysis_key='UPLC', uses_authentic_standard=True,\n",
+    "                              uses_internal_standard=True, is_normalized=True)\n",
     "\n",
     "# Remaining coupling partner (nucleophile)\n",
-    "nucleophile = outcome.products.add() \n",
-    "nucleophile.compound.identifiers.add().CopyFrom(\n",
-    "    reaction_pb2.CompoundIdentifier(value=r'placeholder', type='SMILES')\n",
-    ")\n",
-    "nucleophile.is_desired_product = False\n",
-    "nucleophile.analysis_identity.append('UPLC')\n",
-    "nucleophile.analysis_yield.append('UPLC')"
+    "nucleophile = outcome.products.add(is_desired_product=False) \n",
+    "nucleophile.identifiers.add(value=r'placeholder', type='SMILES')\n",
+    "nucleophile.measurements.add(type='IDENTITY', analysis_key='UPLC', uses_authentic_standard=True)\n",
+    "nucleophile.measurements.add(type='AREA', analysis_key='UPLC', uses_authentic_standard=True)\n",
+    "nucleophile.measurements.add(type='AREA', analysis_key='UPLC', uses_authentic_standard=True,\n",
+    "                             uses_internal_standard=True, is_normalized=True)\n",
+    "\n",
+    "istd = outcome.products.add(is_desired_product=False)\n",
```

Comment:
Related question: is there a reason for the asymmetry in how internal and authentic standards are treated? e.g. should we move all the internal standard stuff to ProductMeasurement instead of having it be part of inputs or workups?

Reply:
Missed this comment earlier somehow...

The reason for adding the internal standard as a product is so we can capture its raw LC peak area. It's not my favorite way to encode that information, but I think it's the most general/flexible. This does have the unfortunate side effect of having the internal standard show up in the list of products. We can strip it out by looking for workup components but there's a risk that users would just look at the list of reactants and list of products. Theoretically, we could have a separate list of `repeated ReactionProduct standards` to parallel `repeated ReactionProduct products` in each reaction outcome? 

@skearnes the asymmetry is just because authentic standards aren't (typically) present in the crude reaction mixture itself, whereas internal standards are often added in solution as part of the input or during a workup.

Reply:
Got it, thanks. Separating into `standards` and `products` seems like a reasonable workaround. @michaelmaser what do you think?

Reply:
Yes, I think that sounds reasonable. Should work as `get_reaction_smiles` only adds from `outcome.products`